### DISCUSSION TITLE
New App Icon Soon?

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [DEMOLITIONDON96/Demolition-Engine#21] [Announcements]
![47E3B934-B516-4EA5-8CAD-EE7A7EDBF9E7](https://user-images.githubusercontent.com/87618246/162480998-7b774226-5843-4aab-9926-352db2b22941.jpeg)

**Comment by charlesisfeline** [comment]
cool icon

**Comment by Theoyeah** [comment]
Cool
